src/urdf_geometry_parser.cpp

```diff
@@ -201,4 +201,18 @@ namespace urdf_geometry_parser{
     }
     return false;
   }
+
+  bool UrdfGeometryParser::getJointAngularSpeedLimit(const std::string& joint_name,
+                                                     double & angular_speed_limit)
+  {
+    if(model_)
+    {
+      urdf::JointConstSharedPtr joint(model_->getJoint(joint_name));
```

Comment:
If the requested joint name is not available within the model,  this method fails. 

(please double check this,  I'm from mobile at the moment)

Reply:
bump